tests/jquery.flot-drawSeries.Test.js

```diff
@@ -386,5 +386,30 @@ describe('drawSeries', function() {
             expect(Array.prototype.slice.call(insideColor2)).toEqual(rgba(0,0,255,1));
             expect(Array.prototype.slice.call(insideColor3)).toEqual(rgba(0,0,255,1));
         });
+
+        it('should use a barWidth based on points distance', function () {
+            var fixture = setFixtures('<div id="demo-container" style="width: 800px;height: 600px">').find('#demo-container').get(0),
+                placeholder = $('<div id="placeholder" style="width: 100%;height: 100%">');
+            placeholder.appendTo(fixture);
+            var testVector = [[[[[0.1, 1], [0.2, 10]]], 0.08],
+                            [[[[1, 1], [2, 10]]], 0.8],
+                            [[[[10, 1], [20, 10]]], 8],
+                            [[[[1000, 1], [2000, 10], [2100, 10]]], 80],
+                            [[[]], 0.8],
+                            [[[[-5, 1], [30, 15], [20, 7], [5, 2]]], 8]],
+                plot;
+            for (var i = 0; i< testVector.length; i++) {
```

Comment:
I'd prefer to see forEach in tests because it is easier to read, but it is OK this way too.